apps/client/src/widgets/community/components/user-comment/user-comment.tsx

```diff
@@ -53,28 +87,44 @@ const UserComment = ({ comment, replyCount, images }: UserCommentProps) => {
       </div>
       {isRepliesOpen && (
         <>
-          {mockUserCommentReplies.map(
+          {allCommentReply.map(
             ({
-              id,
+              commentReplyId,
               profileImage,
-              writerNickName,
+              writerNickname,
               createdAt,
               content,
               images,
-            }) => (
-              <UserCommentReply
-                key={id}
-                profileImage={profileImage}
-                writerNickName={writerNickName}
-                createdAt={getTimeAgo(createdAt)}
-                content={content}
-                images={images}
-              />
-            ),
+              writerId,
+            }) => {
+              if (typeof commentReplyId !== 'number') {
+                return null;
+              }
+
```

Comment:
요게 number 가 아닐 가능성이 있나요.? 
null 일 수 있어서 이렇게 처리하신 걸까유 
이거 꼭 가드가 필요하다면 

```suggestion
  if (commentReplyId == null) {
    return null;
  }
```

요게 조금 더 직관적일 것 같아요!

Reply:
옹.. 좋아요!